packages/kit/src/vite/preview/index.js

```diff
@@ -47,13 +47,13 @@ export async function preview(vite, config, protocol) {
 		// files in `static`
 		vite.middlewares.use(scoped(assets, mutable(config.kit.files.assets)));
 
```

Comment:
we can get rid of this — it's covered by the next middleware now that we're using `publicDir`

```suggestion
```

Reply:
after getting rid of this, there's only one occurrence of `mutable` (for prerendered dependencies), so perhaps we should inline the `sirv` call. though we might then need to ensure that `prerendered/dependencies` always exists, as I think sirv will barf otherwise

Reply:
I think it's nicely structured as a separate method. I took a look at inlining it locally and it doesn't feel quite as clean, so if it's all the same I'd probably just leave it as is